app/webpacker/scripts/autocomplete.js

```diff
@@ -32,7 +32,7 @@ const initAutocomplete = ($el, $input) => {
   const el = document.getElementById($el);
   const inputValueTemplate = result => (typeof result === "string" ? result : result && result.name);
   const suggestionTemplate = result =>
```

Comment:
inputValueTemplate and suggestionTemplate functions can be move out of initAutocomplete.

Reply:
I don't see why you would want to do this? They should be scoped to the function as they are not used elsewhere

Reply:
I agree with @TobyRet, if its not being used else where then it should be scoped to the function.